packages/cli/src/commands/stream/schema/create.ts

```diff
@@ -0,0 +1,102 @@
+import { TileDocument } from '@ceramicnetwork/stream-tile'
+import { flags } from '@oclif/command'
+
+import { Command } from '../../../command'
+import type { CommandFlags } from '../../../command'
+import { parseContent, parseControllers } from '../../../utils'
+
+type Flags = CommandFlags & {
+  determinitic?: boolean
+  'only-genesis'?: boolean
+
+  did?: string
+  controllers?: string
+}
+
+export default class Create extends Command<
+  Flags,
+  {
+    content: string
+  }
+> {
+  static description = 'Create a new Schema'
+
+  static args = [
+    {
+      name: 'content',
+      required: true,
+      description: 'Schema Body',
+    },
+  ]
+  static flags = {
+    ...Command.flags,
+    deterministic: flags.boolean({
+      char: 'd',
+      description:
+        'Document content is created deterministically from the inputs.  This means ' +
+        'that creating a schema document with identical content to an existing schema document ' +
+        'will be a no-op.',
+      default: false,
+    }),
+    'only-genesis': flags.boolean({
+      char: 'g',
+      description: 'Only create the genesis object. No anchor will be created',
+      default: false,
+    }),
+
+    did: flags.string({
+      exclusive: ['key'],
+      description: 'Creator DID',
+    }),
```

Comment:
I don't think that would work

Reply:
I based this off of it's use in `/did/get.ts`. I just tested and confirmed and it doesn't work. Thank you for catching this!